packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -442,7 +443,7 @@ class SystemRulesTable extends React.Component {
         let result;
 
         if (severity.length > 0 && hidePassed) {
-            result = passedRows.filter(row => severityRows.includes(row));
+            result = passedRows.filter(row => severityRows.map((row) => row.cells).includes(row.cells));
```

Comment:
Here's where we combine filtered results. The filtered results, however, will have *different* parent for the children rows. So what we're doing here is making sure we only compare the `cells` and forget about the parent. Combining results worked well before because of the bug - `parent` was the same in any filtered row.